src/main/java/seedu/commands/InvalidCommand.java

```diff
@@ -6,6 +6,6 @@
 public class InvalidCommand extends Command {
     @Override
     public void execute(FinancialTracker finances, Ui ui) {
-        //TODO
+
```

Comment:
Ok so I should add an ui.printInvalid also if you want can help make the dummy function so you can put the method call here.

Reply:
Yeah, I noticed that was missing so I left it blank. Later when we do all the error catching we can do a proper one. As you said, I'll just do a dummy method for now.